src/utils/format.ts

```diff
@@ -12,17 +16,17 @@ const sortObjKeys = (obj) =>
       return acc;
     }, {});
 
-const formatParamsJson = (obj) => JSON.stringify(sortObjKeys(obj));
+const formatParamsJson = (obj: AnyObject): string =>
+  JSON.stringify(sortObjKeys(obj));
 
-const formatOracleGetInt = (resultBn) => {
+const formatOracleGetInt = (resultBn: any): number => {
   const resultBig = new Big(resultBn.toString()).times(new Big('1e-18'));
   try {
     resultBig.constructor.strict = true;
-    const resultNumber = resultBig.toNumber();
-    return resultNumber;
+    return resultBig.toNumber();
   } catch (e) {
     throw Error(
-      `Converting ${resultBig.toString()} to number will result in loosing precision`,
+      `Converting ${resultBig.toString()} to number will result in loosing precision`
     );
   }
 };
```

Comment:
This was there before but I just found it...
`formatOracleGetInt` is a misleading name for the method because it returns a float
rename it to `formatOracleNumber`

Reply:
commit : 23cda40a3321e2bf97d161c568a3e619f6391fdc